src/VisualStudio/Setup/ProvideRoslynBindingRedirection.cs

```diff
@@ -24,7 +24,6 @@ public ProvideRoslynBindingRedirectionAttribute(string fileName)
                 AssemblyName = Path.GetFileNameWithoutExtension(fileName),
                 PublicKeyToken = "31BF3856AD364E35",
                 OldVersionLowerBound = "0.7.0.0",
-                OldVersionUpperBound = "1.3.0.0",
```

Comment:
Not setting this means it'll simply take the upper bound to be equal to the new version. We've already had this change in our Dev15 code for ages.